reach

```diff
@@ -179,16 +197,26 @@ install_docker () {
     printf "RedHat: https://docs.docker.com/engine/install/rhel/\n"
     printf "SLES: https://docs.docker.com/engine/install/SLES/\n"
   fi
-  cd ... || exit
+  cd $PWD
 }
 
 
 for DEP in make curl docker docker-compose; do
   if ! (which "${DEP}" >/dev/null 2>&1); then
-    printf "Reach relies on an installation of %s do you want to install it? [Y,n]: " "$DEP"
+    printf "Reach relies on an installation of ${DEP}; do you want to install it? [Y,n]: "
```

Comment:
Are you not using `shellcheck` on this file? I'm pretty sure it'd complain about string interpolation here.